src/rotation.rs

```diff
@@ -111,9 +111,9 @@ impl<T, B> From<[T; 3]> for EulerAngles<T, B> {
     }
 }
 
-impl<T, B> Into<[T; 3]> for EulerAngles<T, B> {
-    fn into(self) -> [T; 3] {
-        [self.a, self.b, self.c]
+impl<T, B> From<EulerAngles<T, B>> for [T; 3] {
```

Comment:
It look like some generics i think, but it was working for into so it should continue to work like it was.

Reply:
The `Into` impl worked because it was for a type defined here. But the implementation you have in the PR, the type is not defined here. Maybe Rust started being more smart? I don't recall seeing this previously possible.

Reply:
i know that rust create a into impl derive of the from one so maybe it's that.

Reply:
This is expressly permitted by [the orphan rules](https://doc.rust-lang.org/reference/items/implementations.html#orphan-rules) because the type parameter is a local type. IIRC this was relaxed some time in the last couple years.